scripts/xtensa-build-zephyr.py

```diff
@@ -1169,6 +1170,32 @@ def reproducible_checksum(platform, ri_file):
 	chk256 = sof_ri_info.EraseVariables(ri_file, parsed_ri, west_top / repro_output)
 	print('sha256sum {0}\n{1} {0}'.format(repro_output, chk256))
 
+def create_firmware_bundle():
+	if shutil.which('tar') is None:
+		print("tar tool not found, not creating firmware tarball")
+		return
+
+	# vendor directories (typically under /lib/firmware) to include
+        # in the tarball
+	vendor_dirs = [ 'intel']
+
+	build_top = pathlib.Path(west_top, 'build-sof-staging', 'sof')
+	# special case when build script is called for a single
+	# target only
+	if len(args.platforms) == 1:
+		tarball_name = f'sof-%s.tar.gz' % args.platforms[0]
+	else:
+		tarball_name = 'sof.tar.gz'
+
+	if args.pristine:
+		tarball = build_top / tarball_name
+		tarball.unlink(missing_ok = True)
```

Comment:
I don't understand the `if args.pristine` here, I think a new tar file should be created always. This is not a Makefile, this python script has no way to be "smart" and recreate the tarfile only when one of the components has changed, it does not have this information. So it should be reliable and stupid and recreate it always.

Reply:
Ack, I was trying to adjust to the SOF CI builders way of calling the script (one platform at a time), but alas this doesn't a) work, b) make sense, so indeed let's drop this.